lib/parser/index.ts

```diff
@@ -67,7 +69,7 @@ function exportsFn(policy) {
   // ensure we always update the version of the policy format
   data.version = latestVersion();
   // put inline comments into the exported yaml file
-  return addComments(yaml.safeDump(data));
+  return addComments(yaml.dump(data));
```

Comment:
`safeDump` and `saveLoad` have been renamed to `dump` and `load` in v4
https://github.com/nodeca/js-yaml/blob/0d3ca7a27b03a6c974790a30a89e456007d62976/migrate_v3_to_v4.md?plain=1#L10